var/spack/repos/builtin/packages/openimageio/package.py

```diff
@@ -8,14 +8,15 @@
 
 
 class Openimageio(CMakePackage):
-    """OpenImageIO is a library for reading and writing images, and a bunch of
-    related classes, utilities, and applications."""
+    """Reading, writing, and processing images in a wide variety of file formats, using
+    a format-agnostic API, aimed at VFX applications."""
 
     homepage = "https://www.openimageio.org"
```

Comment:
Homepage is a redirect to github which lists https://openimageio.readthedocs.io/, so that may be preferred as a non-github url. You can also add a `git` attribute with the github repository.

Reply:
I added `git` and adjusted `homepage` based on your recommendation.